### PR TITLE
Added chapter parse

### PR DESCRIPTION
**Pull request by brettvanderwerff**
I made this pull request to address issue #6 and change the application so that chapters are parsed alongside the quote. It seems to work at all the endpoints I tried /notes-api,  /notes, /random-note.
Right now the quotes are tied to the chapter as a key value pair in a non-nested dictionary. I can change this structure if you want. I'm new to Github, so hopefully I am doing these pull requests correctly. I'm working though making branches etc. for the first time.  Thanks again for letting me take a crack at some of these issues.  

EDIT:

I could make it structure of dictionaries nested in a list if that seems better:

`...{'notes': [{'quote': 'quote 1' , 'chapter' : 'chapter 13'}, {'quote': 'quote 2' , 'chapter' : 'chapter 76'}]}...`

### CODE REVIEW

File: app/notes.py [hunk 1]
```diff
@@ -24,11 +24,12 @@ def update_notes():
...
         note_tables = soup.select('table table')
-        quotes = []
+        quotes = {}
         for note in note_tables:
             quote_element = note.select('td:nth-of-type(2) span')[0]
-            text = quote_element.contents[0]
-            quotes.append(text)
+            quote_text = quote_element.contents[0]
+            quote_chapter = quote_element.find_previous('h2').contents[0].contents[0]
+            quotes[quote_text] = quote_chapter
```
Comment (mammuth) [inline app/notes.py]: I think we should create dictionaries for every note, but keep `quotes` a list. The resulting structure added to `NOTES_STORE` would then be:
```
[
    {
        'text': 'the quote text here', 
        'chapter': 'the chapter'
    },
    {
        'text': 'another note and its text here', 
        'chapter': 'another chapter'
    },
]
```
Reasoning: The structure is clearer, and we probably never need to lookup a chapter of a note separately, we just expose both in the API and the frontend.

**Review by mammuth — CHANGES_REQUESTED** [review]
Just drop me a line, if you struggle with my feedback. But I guess it's not too hard! 💪

File: app/notes.py (hunk 1 above)
Comment (brettvanderwerff) [reply app/notes.py]: That's actually the same structure I was thinking about doing after I submitted the request if you see my post above. I didn't think about it until after i requested :). I'll go ahead and do that

Comment (mammuth) [reply app/notes.py]: Ah true, read that some hours before I answered and forgot it 😊

Comment (mammuth) [reply app/notes.py]: And your example might be better - `quote` or `note` is probably better than `text`

**Comment by brettvanderwerff** [comment]
Alright, I think I changed the structure to match what you are after. This way is cleaner and actually was easier. Let me know if there is anything that you would like to be refined.

**Review by mammuth — APPROVED** [review]

**Comment by mammuth** [comment]
Looks good now, thanks for the contribution! :tada: :beers:

**Pull request merged**